xcodeproj/internal/xcodeproj_incremental_rule.bzl

```diff
@@ -669,9 +691,16 @@ Are you using an `alias`? `xcodeproj.focused_targets` and \
 
     # Schemes
 
+    autogeneration_config_file = _write_autogeneration_config_file(
+        actions = actions,
+        config = ctx.attr.scheme_autogeneration_config,
+        name = name,
+    )
+
```

Comment:
This should happen in `_write_schemes` (pass in `ctx.attr.scheme_autogeneration_config`).

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/10fe291baf2701603773c923451de58dfdba1156